tests/nsrt_learning/strips_learning/test_oracle_learner.py

```diff
@@ -34,3 +34,36 @@ def test_oracle_strips_learner():
     Delete Effects: [Clear(?block:block), GripperOpen(?robot:robot), On(?block:block, ?otherblock:block)]
     Side Predicates: []
     Option Spec: Pick(?robot:robot, ?block:block)]"""  # pylint: disable=line-too-long
+    # Test with unknown options. Expected behavior is that the operators should
```

Comment:
did you verify that this test was crashing before the change?

Reply:
yup